app/qml/Status/Application/StatusWindow.qml

```diff
@@ -5,7 +5,12 @@ import Qt.labs.settings
 
 import Status.Application
 
-/** Administrative scope
+import Status.Controls.Navigation
```

Comment:
Related to this file, I also like the idea of having a layer on top to manage application events and global actions.